ui/src/app/applications/components/applications-list/applications-tiles.tsx

```diff
@@ -256,6 +257,16 @@ export const ApplicationTiles = ({applications, syncApplication, refreshApplicat
                                                         </div>
                                                         <div className='columns small-9'>{app.spec.destination.namespace}</div>
                                                     </div>
+                                                    <div className='row'>
+                                                        <div className='columns small-3' title='Age:'>
+                                                            Age:
+                                                        </div>
```

Comment:
In the application summary, we use the label `CREATED AT` to show when the resource was created. I think we should be consistent with the label. Would it be okay to change the label `Age` to `Created AT`

Reply:
But `Created at` is a timestamp, while Age is a time delta (as in the screenshot, x hours days etc).

In the tree view, the root node shows the age (and nodes in general have an age label), which makes it easier to observe the resources - as you don't have to do the math now-created_at.

Reply:
In the pod summary we display like this `CREATED AT
12/01/2022 07:15:48 (2 hours ago)`. Can we use this label and format?

Reply:
yes, I'll do that